src/main/game/KeyEventController.java

```diff
@@ -30,174 +32,159 @@ public KeyEventController(Game game, View view, Scene scene) {
-            case DOWN:
-                log.debug("DOWN Key pressed (Make)");
-                this.game.cycleMakeOptionDown();
-                break;
-            case ENTER:
-                log.debug("Enter pressed");
-                if (game.getCurrentType() == UnitEnum.COLONIST) game.executeMakeCommand(UnitEnum.COLONIST, "base");
-                else {
-                    switch (this.game.getCurrentMakeOption()) {
-                        case 0:
-                            game.executeMakeCommand(StructureEnum.BASE, "melee");
-                            break;
-                        case 1:
-                            game.executeMakeCommand(StructureEnum.BASE, "ranged");
-                            break;
-                        case 2:
-                            game.executeMakeCommand(StructureEnum.BASE, "colonist");
-                            break;
-                        case 3:
-                            game.executeMakeCommand(StructureEnum.BASE, "explorer");
-                            break;
-                    }
-
+        if (key == controlReader.getControl("COMMAND_UP")) {
+        	log.debug("UP Key pressed (Make)");
+            this.game.cycleMakeOptionUp();
+            
+        } else if (key == controlReader.getControl("COMMAND_DOWN")) {
+        	log.debug("DOWN Key pressed (Make)");
+            this.game.cycleMakeOptionDown();
+        	
+        } else if (key == controlReader.getControl("SELECT_ITEM")) {
+            log.debug("Enter pressed");
+            if (game.getCurrentType() == UnitEnum.COLONIST) game.executeMakeCommand(UnitEnum.COLONIST, "base");
+            else {
+                switch (this.game.getCurrentMakeOption()) {
+                    case 0:
+                        game.executeMakeCommand(StructureEnum.BASE, "melee");
+                        break;
+                    case 1:
+                        game.executeMakeCommand(StructureEnum.BASE, "ranged");
+                        break;
+                    case 2:
+                        game.executeMakeCommand(StructureEnum.BASE, "colonist");
+                        break;
+                    case 3:
+                        game.executeMakeCommand(StructureEnum.BASE, "explorer");
+                        break;
                 }
-                this.gettingMakeList = false;
-                this.game.setShowingMakeDetails(false);
-                break;
-            default:
-                break;
-        }
 
+            }
+            this.gettingMakeList = false;
+            this.game.setShowingMakeDetails(false);
+        }
     }
 
     public void keyReleased(KeyEvent e) {
         if (this.gettingMoves) {
             this.getMoves(e);
         } else if (this.gettingMakeList) {
-            this.getMakeLIst(e);
-        } else if (e.isControlDown()) {
+            this.getMakeList(e);
+        } else if ((e.isControlDown() && (controlReader.getControl("CHANGE_CYCLING") == KeyCode.CONTROL)) ||
```

Comment:
yeah, but say they map their control key to something else, then its going to mess up our controller. Just saying we have to think about this later.